### ISSUE TITLE
Feedback Aufgabe 09

### ISSUE THREAD

**Issue by MelvinBusch** [maz0o/EIA2#7]
1. `Deklariert wird es mit <HTMLInputElemet>` was ist damit gemeint? Das ergibt keinen Sinn. Was ist "es" und was wird deklariert?
2. Server AD viel zu ungenau. Was passiert denn auf der Server-Seite der Anwendung? Aus `...` werde ich leider nicht schlau
3. Was tut die Funktion `findCallback` und wo finde ich das dazugehörige AD?
4. Selbiges gilt für die Funktion `prepareAnswer()`.
5. Ansonsten in Ordnung

**Comment by mtmatern** [comment]
Zu ... : Es hieß in der Vorlesung, man soll nur für die Sachen ADs machen, die man selber geändert hat. Die ... stehen für die cases, die schon im Code implementiert sind. Auch die function find callback und prepareAnswer ist ja schon drin. Deshalb wurde auch das nicht weiter ausgeführt. Dadurch erklärt sich, dass das Server AD wie in 2. geschrieben, ungenau ist.

**Comment by maz0o** [comment]
Ja, es hieß doch das nur die Dinge die wir im Code ergänzen auch ins AD sollen?
